src/packtPublishingFreeEbook.py

```diff
@@ -203,7 +206,15 @@ def __get_my_all_books_data(self):
                 for fm in self.download_formats:
                     if url.find(fm) != -1:
                         download_urls[fm] = url
-            self.book_data.append({'title': title, 'download_urls': download_urls})
+            order_date = parse(
```

Comment:
`datetime.strptime` won't work here - it depends on locales, that's why I'm using `dateutil.parse.parser`.